jina/proto/jina.proto

```diff
@@ -110,6 +110,19 @@ message Document {
 
     // modality, an identifier to the modality this document belongs to. In the scope of multi/cross modal search
     string modality = 21;
+
+    // List of evaluations performed on the document
+    repeated Evaluation evaluations = 23;
+}
+
+/**
+ * Represents the result of an Evaluation.
+ */
+message Evaluation {
+    // identifier to identify how the evaluation was performed (several evaluations may come together)
+    string id = 1;
```

Comment:
please refer it to `Relevance`, so as a score it can be self-contained and explainable

```proto
message Relevance {
    float value = 1; // value
    string op_name = 2; // the name of the operator/score function
    string description = 3; // text description of the score
    repeated Relevance operands = 4; // the score can be nested
    string ref_id = 5; // the score is computed between doc `id` and `ref_id`
}
```